resolwe_bio/processes/alignment/star.yml

```diff
@@ -305,6 +305,12 @@
           description: |
             Size of the bins, in bases, for the output of the bigwig. Only possible if 'Tool to calculate BigWig'
             is deepTools. If BigWig is calculated by UCSC BedGraphToBigWig then bin size is 1.
+    - name: star_sort
+      label: Sorting with STAR
+      type: basic:boolean
+      default: false
+      description: |
+        Set this to true to force sorting with STAR, which might be slower and more memory demanding.
```

Comment:
Please mention that sorting is done using samtools if false. Maybe something like

> Sort using STAR (true) which may be time and memory intensive or samtools (false)